workshops/templates/forms/trainingrequest.html

```diff
@@ -0,0 +1,38 @@
+{% extends "base_nonav_fixed.html" %}
+
+{% load crispy_forms_tags %}
+{% load selectable_tags %}
+
+{% block content %}
+<p>We run two-day instructor training classes for people who wish to learn how to
```

Comment:
Do you mean sth like that?
![image](https://cloud.githubusercontent.com/assets/1645996/15635677/d501cc6e-25e5-11e6-8c75-014d2b433a7c.png)